typescript/src/schema/proto.ts

```diff
@@ -153,6 +156,18 @@ function pagingResponseType(
 function toInterface(type: string) {
   return type.replace(/\.([^.]+)$/, '.I$1');
 }
+function getMethodComments(
```

Comment:
OK so the next step of making comments a black box is to make this function a public method of `Comments` class. You don't need to expose the comments map, just keep an instance of `comments`, and do `comments.getMethodComments(methodName)` when needed.

Reply:
It totally makes sense to me. I tried to put everything in `comments` class: `getMethodComments` `getServiceComments` and `getCommentsMap`, but I realized that the argument `method: MethodDescriptorProto` requires type from `proto.ts` and `proto.ts` wants to use `comments.ts` which is a circle. 

Or I can pass `method.name` to `getMethodComments`.